app/models/assessment.rb

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+class Assessment < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :format
+  belongs_to :created_by, class_name: 'User'
+
+  enum visibility: %i[internal external]
+
+  has_many :questions
+
+  validates :format, :created_by, presence: true
+
+  swagger_schema :Assessment do
```

Comment:
somewhere else brought up that we might need these swagger definitions to be in the V1 namespace since they are part of the API